app/views/publish/courses/skilled_worker_visa_sponsorship/_form_fields.html.erb

```diff
@@ -6,11 +6,16 @@
       </h1>
     </legend>
 
-   <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
-     Can your organisation sponsor Skilled Worker visas for this course?
-   </legend>
+    <% if @course.pg_teaching_apprenticeship? && !@provider.can_sponsor_skilled_worker_visa %>
+        <p class="govuk-body">Learn more about <%= govuk_link_to "recruiting trainee teachers from overseas", "https://www.gov.uk/guidance/recruit-trainee-teachers-from-overseas-accredited-itt-providers" %>.</p>
+    <% end %>
+
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
```

Comment:
I think the markup for this form is a bit dodgy. I thought I updated it but it looks i only fixed it for the student visas (if you rebase). For this one, i don't think the heading should be wrapped in a legend. Eg https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/views/publish/courses/student_visa_sponsorship/_form_fields.html.erb#L2